SETUP.md

```diff
@@ -30,7 +30,7 @@ This page describes the necessary steps to take before running and using Warden
 Clone the repo:
 
 ```bash
-git clone git@github.com:eq-lab/wardenprotocol.git
+git clone https://github.com/eq-lab/wardenprotocol.git & cd wardenprotocol
```

Comment:
```suggestion
git clone https://github.com/warden-protocol/wardenprotocol.git & cd wardenprotocol
```

Reply:
did you push the changes? I don't see any difference

Reply:
now yes